### PR TITLE
Changed `extract` function to `extract_from_file`

### LINKED ISSUE #26: Rename `extract` function to `extract_from_file`
This is related to #25 to distinguish between a file and bytes.

### PR DESCRIPTION
**Pull request by gabb222**
Fixes #26

### CODE REVIEW

**Comment by gabb222** [comment]
@Theia-Scientific/developers ready for review

File: tspng/extraction.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import json
 import os
 
-def extract(path,mime_type=MIME_TYPE):
+def extract_from_file(path,mime_type=MIME_TYPE):
```
Comment (volks73) [inline tspng/extraction.py:8]: I would add a space after the comma between the two arguments, `path,mime_type`.

**Review by volks73 — APPROVED** [review]
Looks good.

One minor, non-blocking change. You can address in this PR or address later.

File: tspng/extraction.py (hunk 1 above)
Comment (gabb222) [reply tspng/extraction.py]: Made the change as requested

**Pull request merged**